dev/tasks/tasks.yml

```diff
@@ -1392,6 +1392,7 @@ tasks:
         PYTHON: "{{ python_version }}"
         PANDAS: "{{ pandas_version }}"
         NUMPY: "{{ numpy_version }}"
+        PARQUET_REQUIRE_ENCRYPTION: "OFF"
     {% if cache_leaf %}
       # use the latest pandas release, so prevent reusing any cached layers
       flags: --no-leaf-cache
```

Comment:
Just for the record, isn't this condition inverted? In the parameters, `cache_leaf` is true for the fixed pandas version (0.24)...

@kszucs

Reply:
> It is. I suggest to remove `cache_leaf` jinja variable and just set `flags: --no-leaf-cache` for all cases.

I just inverted the check, but can also remove it altogether. I assume the cache is still useful for the one build with fixed pandas version?